images/kubeadm/runner

```diff
@@ -26,7 +26,7 @@ if [ ! -e kubernetes-anywhere ]; then
 
   # Explicitly version this dependency so that upstream commits can't
   # immediately break e2e jobs, and we have control over upgrading/downgrading.
-  git -C kubernetes-anywhere checkout 3ea5e1f14e57c1eb8c21455348d9825f277be350
+  git -C kubernetes-anywhere checkout b5a8c03c3d08b068dc2af6208005b2dd342d1356
 fi
 
 # This is required until https://github.com/kubernetes/kubernetes-anywhere/issues/332
```

Comment:
this issue is closed, should we remove this?

Reply:
@Lion-Wei I think @BenTheElder is referring to [kubernetes-anywhere issue # 332](https://github.com/kubernetes/kubernetes-anywhere/issues/332) being closed (see comment, line 32). So the question is whether the line below that reads the environment variable ${GOOGLE_APPLICATION_CREDENTIALS} can now be removed. I don't see anything in the k-a code that reads in this env variable (that is, nothing obvious in k-a that would be a direct replacement for the lines below here). Maybe @errordeveloper knows more about whether this can be removed?

Reply:
Yes, I don't have anymore insight here but if this is true then we have one less reason to need a custom image.